cmd/apply.go

```diff
@@ -53,5 +53,6 @@ var applyCmd = &cobra.Command{
 func init() {
 	applyCmd.Flags().StringArrayVarP(&InputFiles, "files", "f", []string{}, "Specify files")
 	applyCmd.Flags().StringVarP(&Namespace, "namespace", "n", "", "Namespace or project to deploy your application to")
+	applyCmd.Flags().BoolVar(&EnableValidation, "enable-validation", false, "Enable Validation of input file")
```

Comment:
I thought that we are going to do inverse off this, similar to `kubectl`. 
By default validations are enabled, optionally it can be skipped with `--skip-validation`

Reply:
@kadel, currently schemas are not stable so I did inverse operation

Reply:
problem is that if we do this first then we will have to deprecate it in the future, as we really want validation by default.

Should we really merge this if we know that this Scheme is not stable enough to rely on it? If it can't be trusted it is not useful anyway

Reply:
For example, https://github.com/kedgeproject/kedge/pull/544#issuecomment-355508237

Reply:
Then we need to solve those problems first.  It has to work for most cases before this is merged. 
For corner cases where validation fails, `--skip-validation` can be used before we fix it. But I think it should be enabled by default, otherwise, it is basically useless.

And if we don't trust it enough to be enabled by default, then we should wait with this and fix it before we merge this.